### PR TITLE
Remove claims and classes

### PR DESCRIPTION
**Pull request by muvaf**
### Description of your changes

See https://github.com/crossplane/crossplane/issues/1670 for context.

### Checklist

I have:
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Ensured this PR contains a neat, self documenting set of commits.
- [ ] Updated any relevant [documentation], [examples], or [release notes].
- [ ] Updated the dependencies in [`app.yaml`] to include any new role permissions.

[documentation]: https://github.com/crossplane/crossplane/tree/master/docs
[examples]: https://github.com/crossplane/crossplane/tree/master/cluster/examples
[release notes]: https://github.com/crossplane/crossplane/tree/master/PendingReleaseNotes.md
[`app.yaml`]: https://github.com/crossplane/provider-azure/blob/master/config/package/manifests/app.yaml

### CODE REVIEW

**Comment by muvaf** [comment]
Updated with master versions and manually tested with examples in the repo.

File: pkg/controller/storage/account/account.go [hunk 1]
```diff
@@ -187,7 +187,7 @@ func newAccountSyncDeleter(ao azurestorage.AccountOperations, kube client.Client
 
 func (asd *accountSyncDeleter) delete(ctx context.Context) (reconcile.Result, error) {
 	asd.acct.Status.SetConditions(runtimev1alpha1.Deleting())
-	if asd.acct.Spec.ReclaimPolicy == runtimev1alpha1.ReclaimDelete {
+	if asd.acct.Spec.DeletionPolicy == runtimev1alpha1.DeletionDelete {
```
Comment (negz) [inline pkg/controller/storage/account/account.go:190]: Do we need to handle the case where the DeletionPolicy is not set (and should default to delete, iirc)?

**Review by negz — APPROVED** [review]
I left one comment, and I think the integration tests will need to be disabled, but otherwise LGTM. For the integration tests I've taken an approach of removing them completely in a separate commit so that we can easily revert and update when the new packages are ready.

**Review by hasheddan — APPROVED** [review]

File: pkg/controller/storage/account/account.go (hunk 1 above)
Comment (muvaf) [reply pkg/controller/storage/account/account.go]: Now it deleted if the policy is empty.

**Pull request merged**